### PR TITLE
Add Get/Delete custom rewards

### PR DESCRIPTION
**Pull request by Scorfly**

### CODE REVIEW

File: channels_points.go [hunk 1]
```diff
@@ -17,6 +17,20 @@ type ChannelCustomRewardsParams struct {
 	ShouldRedemptionsSkipRequestQueue bool   `json:"should_redemptions_skip_request_queue"`
 }
 
+// DeleteCustomRewardsParams...
```
Comment (nicklaw5) [inline channels_points.go:20]: Can we remove these comments? They don't add any value. I recently removed all these in a PR. See #103.

File: channels_points.go [hunk 2]
```diff
@@ -17,6 +17,20 @@ type ChannelCustomRewardsParams struct {
...
+type DeleteCustomRewardsParams struct {
+	BroadcasterID string `query:"broadcaster_id"`
+	ID            string `query:"id"`
+}
+
+// GetCustomRewardsParams...
```
Comment (nicklaw5) [inline channels_points.go:26]: Here too.

File: channels_points.go [hunk 3]
```diff
@@ -17,6 +17,20 @@ type ChannelCustomRewardsParams struct {
...
+type GetCustomRewardsParams struct {
+	BroadcasterID         string `query:"broadcaster_id"`
+	ID                    string `query:"id"`
+	OnlyManageableRewards bool   `query:"only_manageable_rewards"`
+}
+
+// ChannelCustomRewards ...
```
Comment (nicklaw5) [inline channels_points.go:33]: Here too.

**Review by nicklaw5 — CHANGES_REQUESTED** [review]

**Comment by Scorfly** [comment]
Yes of course !

Commit amended ;)

**Review by nicklaw5 — APPROVED** [review]
:rocket:

**Pull request merged**